PythonWallpaperChanger/program.py

```diff
@@ -0,0 +1,29 @@
+'''This program extracts some pictures from the NatGeo gallery and store it as local file,
+   then change it as the desktop wallpaper'''
+
+import os
+import ctypes
+import urllib.request as u
+import requests, random
+from datetime import datetime as dt
+
+i='1'
+
+try:
+    day = dt.now().date()
+    day = str(day)
+    temp1 = int(day[-1])
+    tnum = [9000,8900,8930,9010,8920,8970,8950]
+    temp = 2*temp1 - 1
+    t1 = random.choice(tnum)+temp
+    i = str(t1)
+    try:
+        url = 'https://assets-natgeotv.fnghub.com/POD/'+i+'.jpg'
+
+    except:
+        print('Server Error !')
+    path = 'C:\\Users\\Sabi\\Downloads\\m\\pic_dtop\\'+day+'.jpg'
```

Comment:
@amrs-tech  How can someone else access your files? Add them in the repo and update the links. Thanks.

Reply:
@DipanshKhandelwal Sorry for the mistakes. Thanks, I'll correct it.